index.html

```diff
@@ -37,6 +52,9 @@ <h2>Insert Books</h2>
         <button type="submit" id="add-btn">Add</button>
```

Comment:
The _add button_ is not aligned appropriately. Please note that we actually want our design to as much possible follow the [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) Kindly improve this aspect by ensuring that
- [ ] The _add button_ is **aligned to the right** with the form input, as can be seen in the wireframe design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/143276791-875edc43-9138-452f-99ef-56a6f89a62cb.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/143277255-658fcad7-519b-445d-b039-21a2187c6968.png)